website/docs/r/federated_settings_org_config.html.markdown

```diff
@@ -17,12 +17,13 @@ description: |-
 
 ```terraform
 resource "mongodbatlas_federated_settings_org_config" "org_connection" {
-  federation_settings_id     = "627a9687f7f7f7f774de306f14"
-  org_id                     = "627a9683ea7ff7f74de306f14"
-  domain_restriction_enabled = false
-  domain_allow_list          = ["mydomain.com"]
-  post_auth_role_grants      = ["ORG_MEMBER"]
-  identity_provider_id       = "0oad4fas87jL7f75Xnk1297"
+  federation_settings_id            = "627a9687f7f7f7f774de306f14"
+  org_id                            = "627a9683ea7ff7f74de306f14"
+  data_access_identity_provider_ids = ["64d613677e1ad50839cce4db"]
+  domain_restriction_enabled        = false
+  domain_allow_list                 = ["mydomain.com"]
+  post_auth_role_grants             = ["ORG_MEMBER"]
+  identity_provider_id              = "0oad4fas87jL7f75Xnk1297"
```

Comment:
is this more than 20 characters

Reply:
thanks resolved in 31b4e040fbe74a4d67bb34a1227378e56d6c5527